Removes the bio-printer from medical.

Kell-E opened the pull request:
Kills the bio-printer and places a 5u dropper of peridaxon into both of the OR's. 

Also moves a waste disposal cart from OR one slightly to the left because the fact that they weren't symmetrical was bothering me immensely.

https://ibb.co/R4F8pqj

🆑 Kell-E
rscdel: Removes the organ printer (Bio-printer) from OR one.
rscadd: Adds a 5u dropper with peridaxon preset to 1u increments to both OR one and two.
/🆑

Spookerton reviewed (dismissed):
![https://i.imgur.com/fJg9ZgY.png](https://i.imgur.com/fJg9ZgY.png)

SierraKomodo commented on code/modules/reagents/reagent_containers/dropper.dm:
```diff
@@ -108,3 +108,14 @@
 /// Droppers. END
 ////////////////////////////////////////////////////////////////////////////////
+
+/obj/item/reagent_containers/dropper/peridaxon
+	name = "Dropper (Peridaxon)"
+	desc = "Contains peridaxon - used to rescue failing organs."
+
+/obj/item/reagent_containers/dropper/peridaxon/New()
+	..()
+	reagents.add_reagent(/datum/reagent/peridaxon, 5)
+	amount_per_transfer_from_this = 1
+	update_icon()
```
Use `Initialize()` here.

Also, `amount_per_transfer_from_this` can be done in the definition instead of set on init.

```dm
/obj/item/reagent_containers/dropper/peridaxon
	name = "Dropper (Peridaxon)"
	desc = "Contains peridaxon - used to rescue failing organs."
    amount_per_transfer_from_this = 1

/obj/item/reagent_containers/dropper/peridaxon/Initialize()
	..()
	reagents.add_reagent(/datum/reagent/peridaxon, 5)
	update_icon()
```

SierraKomodo replied:
Use `Initialize()` here.

Also, `amount_per_transfer_from_this` can be done in the definition instead of set on init.

```dm
/obj/item/reagent_containers/dropper/peridaxon
	name = "Dropper (Peridaxon)"
	desc = "Contains peridaxon - used to rescue failing organs."
	amount_per_transfer_from_this = 1

/obj/item/reagent_containers/dropper/peridaxon/Initialize()
	. = ..()
	reagents.add_reagent(/datum/reagent/peridaxon, 5)
	update_icon()
```

SierraKomodo commented on code/modules/reagents/reagent_containers/dropper.dm:
```diff
@@ -108,3 +108,14 @@
+	amount_per_transfer_from_this = 1
+
+/obj/item/reagent_containers/dropper/peridaxon/Initialize()
+	..()
```
To fix your test

```suggestion
	. = ..()
```

Kell-E replied:
It is very mad about dots.
Not sure what the expected usage of ..() versus . = ..() so I can't really do much to fix it right now.

SierraKomodo replied:
`..()` runs the parent without affecting any return values. `. = ..()` runs the parent then takes the parent's return value and assigns it to the child proc.

Initialize() expects to receive a valid return value.

SierraKomodo replied:
`..()` runs the parent without affecting any return values. `. = ..()` runs the parent then takes the parent's return value and assigns it to the child proc.

Initialize() expects to receive a valid return value.

SierraKomodo commented on code/modules/reagents/reagent_containers/dropper.dm:
```diff
@@ -108,3 +108,14 @@
+	amount_per_transfer_from_this = 1
+
+/obj/item/reagent_containers/dropper/peridaxon/Initialize()
+	 . = ..()
```
You have an extra space before the initial `.` here causing your indentation error.

Kell-E committed:
Removes the bio-printer from medical.

Kills the bio-printer and places a 5u dropper of peridaxon into both of the OR's. 

Also moves a waste disposal cart from OR one slightly to the left because the fact that they weren't symmetrical was bothering me immensely. 
a


Test fix


test fix fix


fix fix fix

MistakeNot4892 commented:
Should kill the organ_printer class entirely while you're in the area.

SierraKomodo commented:
> Should kill the organ_printer class entirely while you're in the area.

It's still used on some other maps. A notable one that would probably be fine with keeping the printer is skrellship.
